platform/security/certificate/src/main/java/org/codice/ddf/certificate/JKSManager.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.certificate;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Enumeration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JKSManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(JKSManager.class);
+
+    /*---------------------------------------------------------------------
+    |  Method: saveCertToJKSKeyStore
+    |
+    |  Purpose:  Saves a certificate to an existing keystore file,
+    |
+    |  Pre-condition:  A valid keystore file
+    |
+    |  Post-condition: The keystore file will contain the certificate passed in a parameter (certficateChain)
+    |
+    |  Parameters:
+    |        String  keyStorePath    -- path to the keyStore
+    |        String  keyStorePassword   -- password for the keystore
+    |        String  alias           -- alias for the new keystore entry
+    |        String keyPassword      -- password to protect the key
+    |        X509Certificate[] certChain -- the certificate chain for the corresponding public key
+    |        PrivateKey privKey      -- privateKey to be associated with the alias
+    |
+    |  Returns:  None
+    *-------------------------------------------------------------------*/
+
+    public static void addCertToKeyStore(String keyStorePath, String keyStorePassword, String alias,
+            String keyPassword, X509Certificate[] certChain, PrivateKey privKey) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
```

Comment:
We should be using System.getProperty("javax.net.ssl.keyStoreType") here instead of hardcoding to JKS